include/exadg/functions_and_boundary_conditions/interface_coupling.h

```diff
@@ -145,13 +109,101 @@ class InterfaceCoupling
       }
 
       array_solution_dst.resize(array_q_points_dst.size(), dealii::Tensor<rank, dim, double>());
+    }
+
+    // finally, give boundary condition access to the data
+    for(auto boundary : map_bc_)
+    {
+      boundary.second->set_data_pointer(map_vector_index, map_solution);
+    }
+  }
+
+  std::vector<quad_index> const &
```

Comment:
```suggestion
  std::vector<quad_index_type> const &
```

Reply:
But please indicate somewhere/somehow that it is a type. In the same file the name is also used as variable name:

https://github.com/nfehn/exadg/blob/0d2cb65d0267db4318ecdcd4aba83986d1424c47/include/exadg/functions_and_boundary_conditions/interface_coupling.h#L201